tests/unit/conditions/test_time_condition.py

```diff
@@ -27,14 +27,6 @@ def test_invalid_time_condition():
             method="time_after_time",
         )
 
-    # chain id not permitted
```

Comment:
and again - the difference is that rather than it being "not permitted" it's just "not supported"